onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java

```diff
@@ -127,6 +128,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
         mgr.initLoader(ARRIVALS_LIST_LOADER, getArguments(), this).forceLoad();
     }
 
+    @Override
```

Comment:
If `onAttach()` is just calling `super.onAttach()`, could you please just delete this entire method here?